frontend/common/providers/useWebhooks.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  useCreateWebhookMutation,
```

Comment:
Although this simplifies syntax I'm not sure this file is necessary, we don't have this sort of pattern elsewhere - can just use the useCreate etc directly